Tests/Engine/InvokeScriptAnalyzer.tests.ps1

```diff
@@ -468,33 +468,34 @@ Describe "Test CustomizedRulePath" {
 
 Describe "Test -Fix Switch" {
 
-    BeforeEach {
-        $initialTestScript = Get-Content $directory\TestScriptWithFixableWarnings.ps1 -Raw
+    BeforeAll {
+        $scriptName = "TestScriptWithFixableWarnings.ps1"
+        $testSource = join-path $PSScriptRoot ${scriptName}
```

Comment:
I think `${scriptName}` can be simplified to `$scriptName`

Reply:
the "${<name>}" is the formal identifier of the variable. I should have done it to the first variable too. You'll start to see more of these in my PRs. I should note that this is much different than `"$($variable)"` which is definitely unneeded

Reply:
I totally agree that braces make sense inside a string for string interpolation and is better than `"$($variable)"` but can you briefly explain please why one would use curly braces for variables __outside__ of strings (the only use case that I found [here](https://stackoverflow.com/a/36774315) is when the variable name itself contains special characters) or is this just a question of style?

Reply:
in the case above it's inconsistent (since it's not used with `$PSScriptRoot`. my _preference_ is to use braces all the time (sort of like full commands vs aliases), and it's mostly habit. When I'm in a hurry, I sometimes omit them. I think it's especially useful in the middle of strings.